src/modules/paymentwall/views/order_summary.tpl

```diff
@@ -4,7 +4,6 @@
 </h1>
```

Comment:
Looks like you just create new files and forgot to remove old files from old folder structure.